woocommerce/pinterest-for-woocommerce
# Use `-jsdoc` plugins from woocommerce/grow

_This PR requires https://github.com/woocommerce/grow/pull/13_

### Changes proposed in this Pull Request:
Use `-jsdoc` plugins from woocommerce/grow,
instead of maintaining a local copy.

### Screenshots:




### Detailed test instructions:


1. Remove a few lines from https://github.com/woocommerce/pinterest-for-woocommerce/blob/develop/TRACKING.md?plain=1#L17-L22 to see that the result is generated
2. ```sh
   npm i
   npm run doc:tracking
   ```
3. Check that `TRACKING.md` is generated correctly


### Additional details:


### Changelog entry
